images/build-integration/entrypoint.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
+
+# Vars
+BINARY=integration.test
+if [[ ${PLATFORM} == 'windows' ]]; then
+    BINARY=integration.test.exe
+fi
+BINARY_PATH="/opt/crc/bin/${PLATFORM}-amd64/${BINARY}"
+
+# Results
+RESULTS_PATH="${RESULTS_PATH:-/output}"
+
+
+if [ "${DEBUG:-}" = "true" ]; then
+    set -xuo 
+fi
+
+# Validate conf
+validate=true
+[[ -z "${TARGET_HOST+x}" ]] \
+    && echo "TARGET_HOST required" \
+    && validate=false
+
+[[ -z "${TARGET_HOST_USERNAME+x}" ]] \
+    && echo "TARGET_HOST_USERNAME required" \
+    && validate=false
+
+[[ -z "${TARGET_HOST_KEY_PATH+x}" && -z "${TARGET_HOST_PASSWORD+x}" ]] \
+    && echo "TARGET_HOST_KEY_PATH or TARGET_HOST_PASSWORD required" \
+    && validate=false
+
+[[ -z "${PULL_SECRET_FILE_PATH+x}" ]] \
+    && echo "PULL_SECRET_FILE_PATH required" \
+    && validate=false
+
+[[ $validate == false ]] && exit 1
+
+# Define remote connection
+REMOTE="${TARGET_HOST_USERNAME}@${TARGET_HOST}"
+if [[ ! -z "${TARGET_HOST_DOMAIN+x}" ]]; then
+    REMOTE="${TARGET_HOST_USERNAME}@${TARGET_HOST_DOMAIN}@${TARGET_HOST}"
+fi
+
+# Set SCP / SSH command with pass or key
+NO_STRICT='-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null'
+if [[ ! -z "${TARGET_HOST_KEY_PATH+x}" ]]; then
+    SCP="scp -r ${NO_STRICT} -i ${TARGET_HOST_KEY_PATH}"
+    SSH="ssh ${NO_STRICT} -i ${TARGET_HOST_KEY_PATH}"
+else
+    SCP="sshpass -p ${TARGET_HOST_PASSWORD} scp -r ${NO_STRICT}" \
+    SSH="sshpass -p ${TARGET_HOST_PASSWORD} ssh ${NO_STRICT}"
+fi
+
+echo "Copy resources to target"
+# Create execution folder 
+EXECUTION_FOLDER="/Users/${TARGET_HOST_USERNAME}/crc-integration"
+if [[ ${PLATFORM} == 'linux' ]]; then
+    EXECUTION_FOLDER="/home/${TARGET_HOST_USERNAME}/crc-integration"
+fi
```

Comment:
What about windows does the `EXECUTION_FOLDER="/Users/${TARGET_HOST_USERNAME}/crc-integration"` works for windows also?

Reply:
yes, both `powershell` and `pwsh` admit this